pkg/adapter/adapter_test.go

```diff
@@ -450,3 +495,50 @@ func sinkAccepted(writer http.ResponseWriter, req *http.Request) {
 func sinkRejected(writer http.ResponseWriter, _ *http.Request) {
 	writer.WriteHeader(http.StatusRequestTimeout)
 }
+
+func TestAdapter_VhostHandler(t *testing.T) {
+	brokers := "amqp://localhost:5672"
```

Comment:
I think it would be more golang stylish to rewrite these tests as table driven tests.
Kind of like this:
https://github.com/knative-sandbox/eventing-rabbitmq/blob/main/pkg/reconciler/broker/resources/exchange_test.go#L20

so the struct and first two tests might look like this:
```
	for _, tt := range []struct {
		name      string
		brokers string
		vhost      string
		want       string
		}{{
		   name:      "no vhost",
		   brokers: "amqp://localhost:5672",
		   vhost: "",
		   want: "amqp://localhost:5672",
	  },{
		   name:      "vhost",
		   brokers: "amqp://localhost:5672",
		   vhost: "test-vhost",
		   want: "amqp://localhost:5672/test-vhost",
	}} {
	       t.Run(tt.name, func(t *testing.T) {
	             t.Parallel()
	             got := VhostHandler(tt.brokers, tt.vhost)
	             if got != tt.want {
	                   t.Errorf("Unexpected result, want %q got %q", tt.want, got)
	             }
                 })
           }

Reply:
:o nice format, cool! fixed!